float64 x_position
float64 y_position
float64 velocity
float64 yaw
float64 yaw_rate
